gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -0,0 +1,202 @@
+          <li className="nx-list__item">
+            A navigation link section which display an icon plus text in the open state and an icon in the closed
+            state
+          </li>
+        </ul>
+        <NxP>
+          Because <NxCode>NxGlobalSidebar</NxCode> contains product branding it should not be used in conjunction
+          with <NxCode>NxPageHeader</NxCode> or <NxCode>NxNexusPageHeader</NxCode>.
+        </NxP>
+        <NxP>
+          Additional custom content may be added below the navigation links if desired, care should be taken that such
+          content can adapt to both the open and closed states.
+        </NxP>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <h3 className="nx-h3"><NxCode>NxGlobalSidebar</NxCode> Props</h3>
+          </NxTile.SubsectionHeader>
+          <NxTable>
+            <NxTable.Head>
+              <NxTable.Row>
+                <NxTable.Cell>Prop</NxTable.Cell>
+                <NxTable.Cell>Type</NxTable.Cell>
+                <NxTable.Cell>Required</NxTable.Cell>
+                <NxTable.Cell>Details</NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Head>
+            <NxTable.Body>
+              <NxTable.Row>
+                <NxTable.Cell>isOpen</NxTable.Cell>
+                <NxTable.Cell>Boolean</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>This value determines whether the sidebar is open or closed.</NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>toggleOpenIcon</NxTable.Cell>
+                <NxTable.Cell>IconDefinition</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  An icon used by the sidebar's open/close toggle when the sidebar is in the open state. The icon
+                  should make it clear that clicking it will close the sidebar.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>toggleClosedIcon</NxTable.Cell>
+                <NxTable.Cell>IconDefinition</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  An icon used by the sidebar's open/close toggle when the sidebar is in the closed state. This icon
+                  should make it clear that clicking it will open the sidebar.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>logoImg</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell>
+                  The path to an image placed in the top left corner which displays branding and product name.
+                  <strong>Note:</strong> the image should not be larger than 145px wide and 32px tall. It should
```

Comment:
Minor: I typically use alerts for this kind of callout

Reply:
> Minor: I typically use alerts for this kind of callout

Or if you do keep this the way it is, there's a whitespace issue around the `<strong>` that at least needs to be fixed.

Reply:
> Minor: I typically use alerts for this kind of callout

The alert is better: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/3a85d6147904a438c5742677e401a7c9797ec515